src/components/AddBook.js

```diff
@@ -17,8 +17,7 @@ const AddNewBook = () => {
     // eslint-disable-next-line camelcase
     const item_id = Math.random();
     const category = 'Crime';
-    const author = 'A.k Khan';
-    /*     dispatch(addBook(id, title, author)); */
+    const author = 'A.K Khan';
     dispatch(addBooksAsync({
       // eslint-disable-next-line camelcase
```

Comment:
- [ ] Your design is good however kindly consider removing the outer white space, so your page is taking the whole space

|Your design|How it should be|
|-|-|
|![Screenshot_20220715_175655](https://user-images.githubusercontent.com/25227022/179272702-ee399fbe-6e5e-4385-9062-791962c1adbd.png)|![Screenshot_20220715_180244](https://user-images.githubusercontent.com/25227022/179272840-b3575989-15ce-4309-aec3-b1a4dffc9c5c.png)|

Reply:
Thanks